Server 10.13.2 Release Notes

pako81 opened the pull request:
These are the release notes for ownCloud Server 10.13.2.

EParzefall commented on modules/ROOT/pages/server_release_notes.adoc:
```diff
@@ -43,6 +44,34 @@
+
+=== Allow Media Viewer App to work with HEIC and HEIF formats ===
+
+We removed the regular expression related to HEIC and HEIF files from the Preview Manager list as this was preventing the files_mediaviewer app from correctly interpreting such mime types. The app is now able to open such files. https://github.com/owncloud/core/pull/40990[#40990]
```
```suggestion
We removed the regular expression related to HEIC and HEIF files from the preview manager list as this was preventing the files_mediaviewer app from correctly interpreting these mime types. The app is now able to open such files. https://github.com/owncloud/core/pull/40990[#40990]
```

EParzefall commented on modules/ROOT/pages/server_release_notes.adoc:
```diff
@@ -43,6 +44,34 @@
+
+We removed the regular expression related to HEIC and HEIF files from the Preview Manager list as this was preventing the files_mediaviewer app from correctly interpreting such mime types. The app is now able to open such files. https://github.com/owncloud/core/pull/40990[#40990]
+
+=== Check If Account Creation Time Exists For Migrations ===
```
```suggestion
=== Check if Account Creation Time Exists for Migrations ===
```

pako81 commented:
Converting to draft for now to avoid merging as we just discovered another PR could eventually be included.

phil-davis commented on modules/ROOT/pages/server_release_notes.adoc:
```diff
@@ -43,6 +44,34 @@
+
+=== Check if Account Creation Time Exists for Migrations ===
+
+In some rare scenarios it could have happened that the migration responsible for adding the creation time in the oc_accounts table was not correctly inserted into the oc_migrations one with the consequence that it was reattempted i.e. when upgrading apps, even if the column was already present. This has been now fixed. https://github.com/owncloud/core/pull/40991[#40991]
```
```suggestion
In some rare scenarios it could have happened that the migration responsible for adding the creation time in the oc_accounts table was not correctly inserted into the oc_migrations one with the consequence that it was reattempted i.e. when upgrading apps, even if the column was already present. This has been fixed. https://github.com/owncloud/core/pull/40991[#40991]
```

"been now fixed" sounds strange, "now been fixed" is the usual word order. But I don't think that the word "now" adds any useful value, so may as well remove it.

jnweiger reviewed (approved):
Looks good. Thanks!

pako81 commented:
Documented https://github.com/owncloud/core/pull/41014, moved https://github.com/owncloud/core/pull/40974 to the `Solved Known Issues` section and added `Updated App Versions` section. It should be ready now.

mmattel commented on modules/ROOT/pages/server_release_notes.adoc:
```diff
@@ -43,6 +44,50 @@
+
+=== Solved Known Issues
+
+All xref:server_release_notes.adoc#known-issues-10-13-1[known issues] in Server 10.13.1 have been fixed:
```
```suggestion
All xref:known-issues-10-13-1[known issues] in Server 10.13.1 have been fixed:
```
In-doc references do not need to have the page extra stated.

mmattel commented on modules/ROOT/pages/server_release_notes.adoc:
```diff
@@ -43,6 +44,50 @@
+
+Find below a list of updated apps in comparison with the 10.13.1 complete bundle. More information on the changes can be found in the respective changelogs on ownCloud Marketplace.
+
+* https://marketplace.owncloud.com/apps/wopi [Microsoft Office Online]: 1.8.1
```
```suggestion
* {oc-marketplace-url}/apps/wopi [Microsoft Office Online]: 1.8.1
```

mmattel reviewed (approved):
LGTM 👍
